game_patch/rf/ai.h

```diff
@@ -212,6 +212,9 @@ namespace rf
         float movement_radius;
         float movement_height;
         bool use_custom_attack_range;
+#ifdef DASH_FACTION
+        bool explosive_last_damage;
```

Comment:
I'm worried that it will be uninitialized in some cases. Would be best to avoid adding fields but if it's necessary, they should probably be initialized in `ai_init_ai_info`

Reply:
I did spend a lot of time looking for a way to avoid adding a field and wasn't able to find one - I believe it is necessary.

I will work on initializing the new field in `ai_init_ai_info`